core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -100,8 +100,8 @@ UNWIND(	.fnstart)
 	movt r0, #0x0002
 	write_nsacr r0
 
-	movw r0, #0x0000
-	movt r0, #0x0001
+	movw r0, #0x0001
```

Comment:
@dryvenn, i was wondering the same about other platforms.
I think it's better to update the whole source new. If we don't do it now, we will do it later.
Yet, let's see what @jenswi-linaro and @jforissier thinks about it.